src/Components/Admins/Form/form.module.css

```diff
@@ -0,0 +1,34 @@
+.container {
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+label {
+  color: #002147;
+  font-size: 20px;
+  font-weight: bold;
+  margin: 5px 0;
+}
+
+input {
+  border: 1px solid #BDBDBD;
+  border-radius: 4px;
+  padding: 12px;
+  width: 270px;
+}
+
+button {
```

Comment:
do not use pure tags

Reply:
Removed all pure tags and replaced by classes